### DISCUSSION TITLE
Installer script for FrankenPHP binary

### DISCUSSION THREAD

**Discussion by alexander-schranz** [php/frankenphp#1036] [Ideas]
I think FrankenPHP is ideal software to get started with PHP fast even without having docker.

```bash
wget https://github.com/dunglas/frankenphp/releases/download/v1.2.5/frankenphp-mac-arm64
mv frankenphp-mac-arm64 frankenphp
chmod +x frankenphp
```

then already can use it with `./frankenphp php-server --listen 127.0.0.1:8081 --root public`

Maybe we can provide a bash script which downloads the correct binary by detect the cpu architecture and download it to the directory and make it runable.

Such oneliner to install frankenphp into the current directly could look like this:

```bash
curl -sL https://frankenphp.dev/installer.sh | bash -
# or
wget -O - https://frankenphp.dev/installer.sh | bash -
```

Maybe also a parameter for a specific version can be added.

**Comment by withinboredom** [comment]
Like this: https://github.com/dunglas/frankenphp/pull/594

**Reply by alexander-schranz** [reply]
Looks something like that :)
